custom/icds_reports/sqldata/base.py

```diff
@@ -8,12 +8,14 @@
 from sqlalchemy.dialects import postgresql
 
 from corehq.apps.reports.sqlreport import DatabaseColumn, SqlData
+from corehq.sql_db.connections import ICDS_UCR_CITUS_ENGINE_ID, ICDS_UCR_ENGINE_ID
+from corehq.sql_db.routers import forced_citus
 from corehq.toggles import ICDS_COMPARE_QUERIES_AGAINST_CITUS, NAMESPACE_OTHER
 from custom.icds_reports.utils.tasks import call_citus_experiment
 
 
 class IcdsSqlData(SqlData):
-    engine_id = 'icds-ucr'
+    engine_id = ICDS_UCR_CITUS_ENGINE_ID if forced_citus() else ICDS_UCR_ENGINE_ID
```

Comment:
Doesn't this get used by all UCRs as well, which mean we are flipping citus for UCRs too with this? Or does this engine_id get overridden by UCRs?

Reply:
pretty sure this is just used by `sqlagg` based dashboard views

Reply:
👍